comm.go

```diff
@@ -0,0 +1,61 @@
+package cleisthenes
+
+import (
+	"net"
+	"time"
+
+	"github.com/DE-labtory/cleisthenes/pb"
+	"google.golang.org/grpc"
+)
+
+type ConnHandler func(conn Connection)
+type ErrHandler func(err error)
+
+type GrpcServer struct {
+	connHandler ConnHandler
+	errHandler  ErrHandler
+	ip          string
+	lis         net.Listener
+	meta        map[string]string
+}
+
+func NewServer(meta map[string]string) *GrpcServer {
+	return &GrpcServer{
+		meta: meta,
+	}
+}
+
+func (s GrpcServer) MessageStream(streamServer pb.StreamService_MessageStreamServer) error {
+	panic("implement me w/ test case :-)")
+}
+
+func (s *GrpcServer) OnConn(handler ConnHandler) {
+	panic("implement me w/ test case :-)")
+}
+
+func (s *GrpcServer) OnErr(handler ErrHandler) {
+	panic("implement me w/ test case :-)")
+}
+
+func (s *GrpcServer) Listen(ip string) {
+	panic("implement me w/ test case :-)")
+}
+
+func (s *GrpcServer) Stop() {
+	panic("implement me w/ test case :-)")
+}
+
+type DialOpts struct {
```

Comment:
what is purpose of this `DialOpts` struct?

Reply:
with DialOpts we can set option for dial

In practical, one of DialOpts is dial timeout !